jaqpotpy/helpers/helpers.py

```diff
@@ -87,6 +87,7 @@ def create_molecular_req(model, title, description, type):
     independentFeatures.append("Smiles")
     if model.external_feats:
         independentFeatures.append(model.external_feats)
+        independentFeatures = [item for sublist in independentFeatures for item in (sublist if isinstance(sublist, list) else [sublist])]
```

Comment:
We redefine the independent features just after appending to them? Let's remove line 89. 

Also the code here is a bit confusing, I'd rather expose this to a function that explains what we're doing, e.g. 

```python
def flatten_features(features):
    """
    Flattens a list of features where each feature can either be an individual item or a list of items.

    Args:
        features (list): A list of features, where each feature can be an individual item or a list of items.

    Returns:
        list: A flattened list of features.
    """
    return [item for sublist in features for item in (sublist if isinstance(sublist, list) else [sublist])]

# Example usage
independentFeatures = flatten_features(independentFeatures)
```